courses/free-courses-hi.md

```diff
@@ -109,6 +109,7 @@
 * [Java Programming Tutorial (HINDI/URDU)](https://www.youtube.com/playlist?list=PLiOa6ike4WAHljIOitb3vR0nXQgneUedR) - Vikas Pandey, Easytuts4you
 * [Java Tutorial](https://youtube.com/playlist?list=PLX9Zi6XTqOKQ7TdRz0QynGIKuMV9Q2H8E) - Saurabh Shukla Sir
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
+* [Java Tutorials For Beginners](https://www.codewithharry.com/videos/java-tutorials-for-beginners-1/) - CodeWithHarry
```

Comment:
Same course as the entry below. He has embedded it in his site

Reply:
but it contains notes in a form of blog as well and at the bottom handwritten notes and video lectures are there

so its more helpfull than previous one